collector/lambdacomponents/default.go

```diff
@@ -20,6 +20,12 @@ import (
 	"go.opentelemetry.io/collector/exporter/loggingexporter"
 	"go.opentelemetry.io/collector/exporter/otlpexporter"
 	"go.opentelemetry.io/collector/exporter/otlphttpexporter"
+	"go.opentelemetry.io/collector/processor/attributesprocessor"
+	"go.opentelemetry.io/collector/processor/filterprocessor"
+	"go.opentelemetry.io/collector/processor/memorylimiter"
+	"go.opentelemetry.io/collector/processor/probabilisticsamplerprocessor"
+	"go.opentelemetry.io/collector/processor/resourceprocessor"
+	"go.opentelemetry.io/collector/processor/spanprocessor"
```

Comment:
From what I understand, we don't need to add `processorhelper` because the many of these processors depend on it. The `go build` command from `make build` will pull in anything necessary so we don't need to write it out explicitly.